pages/blog/_slug.vue

```diff
@@ -23,8 +23,7 @@
           <time class="tiny">{{ ( new Date(post.fields.publishDate)).toDateString() }}</time>
           <h1>{{ post.fields.title }}</h1>
         </div>
-        <div class="copy">
-          <vue-markdown>{{post.fields.body}}</vue-markdown>
+        <div class="copy" v-html="marked(post.fields.body)">
```

Comment:
Personally, I like the component approach better.

```
<div class="copy">
  <markdown>{{post.fields.body}}</markdown>
</div>
```

Because then I don't have to remember `v-html` and `marked`.
What do you think of that?

Reply:
I agree with you. It is much more elegant to be able to wrap the post in a component. 

I'm not sure off the top of my head the best way to refactor the `filter` approach into a `component` approach. But I'll happily look into it. Maybe running the filter on the `slot`. If you have any ideas let me know.

Reply:
@lab27 Yeah, for me a [single slot](https://vuejs.org/v2/guide/components.html#Single-Slot) with the same logic (`marked(post.fields.body)`) would work for me. :)